src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsBtfBlocking.java

```diff
@@ -25,62 +25,59 @@
       dataProvider = "delayBtf",
       groups = "AdsBtfBlockingOasis"
   )
-  public void adsAtfDelayBtfOasis(String wikiName, String article, boolean isWgVarOn)
-      throws InterruptedException {
-    PageObjectLogging.log("$wgAdDriverDelayBelowTheFold", String.valueOf(isWgVarOn), true);
+  public void adsAtfDelayBtfOasis() {
+    PageObjectLogging.log("$wgAdDriverDelayBelowTheFold", String.valueOf(true), true);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "$wgAdDriverDelayBelowTheFold" 6 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)